lib/cli/line-processor.js

```diff
@@ -68,11 +68,11 @@ module.exports = class LineProcessor {
     const [host, portString] = utils.split(authority, ':', 2, true);
     if (!host) {
       return callback(reportMissingArgument('Host'));
-    } else if (!portString) {
+    } else if (scheme !== 'ipc' && !portString) {
```

Comment:
@belochub just declare port before `if (scheme === 'ipc')` check and then put lines 71-77 under that `if`.
```javascript
const port = 0;
if (scheme !== 'ipc') {
  if (!portString) {
    return callback(reportMissingArgument('Port'));
  }
  port = Number(portString);
  if (isNaN(port) || port < 0 || port >= 65536) {
    return callback(new Error(`Port has incorrect value: ${portString}`));
  }
}
```